CONTRIBUTING.md

```diff
@@ -11,9 +11,32 @@ The code code is mainly written in PHP using the Symfony framework with Twig tem
 
 With an account on [GitHub](https://github.com) you will be able to [fork this repository](https://github.com/MbinOrg/mbin) and `git clone` the repository locally if you wish.
 
-\_Note:\_If you are a contributor with GitHub org rights, you do not need to fork the project, instead you are allowed to use branches.
+_Note:_ If you are a contributor with GitHub org rights, you do not need to fork the project, instead you are allowed to use branches.
 
-Always [create a new Pull Request](https://github.com/MbinOrg/mbin/pulls) in GitHub. Be sure to avoid regression, see below for more info about the coding style as well as testing. All tests in the Pull Request should be green!
+### Way of Working
+
+See below our agreements / guidelines how we work together in the project (in random order):
+
+- Respect each other naturally.
+- [Discuss](https://matrix.to/#/#mbin:melroy.org) (big) issues/changes in the community first and have some consensus (WIP: we are still looking into some voting tool that might help here).
+- Always [create a new Pull Request](https://github.com/MbinOrg/mbin/pulls) in GitHub. Thus feature/topic branches are allowed (despite what C4 spec says). Eg. Do _NOT_ merge directly to the `main` branch!
```

Comment:
If I understand this correctly, the meaning of the last sentence is: "Do _NOT_ merge directly to the main branch without a PR!" Is that right?

Reply:
Ah yes, I need to clear this up. Of course we can merge into the main branch, but the point was to always use PRs.

Reply:
I extend the C4 spec to make this all clear.